### ISSUE TITLE
Missing Docker Image in public registry for the Specified Version in manifest 

### ISSUE THREAD

**Issue by adiii717** [wazuh/wazuh-kubernetes#402]
Is the absence of the referenced image in the public Docker registry intentional or does it indicate a issue with the pipeline?

```
Failed to pull image "wazuh/wazuh-dashboard:4.7.0": rpc error: code = Unknown desc = reading manifest 4.7.0 in docker.io/wazuh/wazuh-dashboard: manifest unknown
```

https://github.com/wazuh/wazuh-kubernetes/blob/master/wazuh/indexer_stack/wazuh-indexer/cluster/indexer-sts.yaml#L62

**Comment by Rishang** [comment]
@adiii717 this might help

https://hub.docker.com/r/wazuh/wazuh-manager/tags

**Comment by rauldpm** [comment]
Hello @adiii717, keep in mind that you are using the `master` branch, which is a development branch (currently referring to `4.8.0`), if you want to deploy please keep in mind that the latest version available at the moment is `4.5.0` and you should refer to the corresponding tag:

- https://github.com/wazuh/wazuh-kubernetes/tree/v4.5.0

For more information, please read the documentation to deploy Wazuh using Kubernetes: https://documentation.wazuh.com/current/deployment-options/deploying-with-kubernetes/kubernetes-deployment.html

**Comment by rauldpm** [comment]
Issue closed due to inactivity
